spec/spec_helper.rb

```diff
@@ -6,14 +6,18 @@
   Knapsack::Adapters::RSpecAdapter.bind
 end
 
-RSPEC_RUNNING_IN_PARALLEL = ENV['PARALLEL_PID_FILE'].nil?
+RSPEC_RUNNING_IN_PARALLEL = ENV['PARALLEL_PID_FILE'].present?
```

Comment:
I think this logic was flipped before.... if the PID file is nil, that means we're running serial/solo/not parallel